docs/CONTRIBUTING.md

```diff
@@ -41,3 +41,25 @@ in working on, it's time to get into the technical foundations of the project. A
 our technical documentation can be found in the [docs/tech](../docs/tech) directory
 where the [README](./tech/README.md) is the best place to get started.
 
+
+# 3 - Developer Expecations
+
+To ensure software reliability and good practices, all contributors must adhere to the following principles.
+
+1. All PRs must be approved 
+   1. Be approved by at least 2 people.
+   2. Must be approved by [Elham Islam](https://github.com/Lham42).
+   3. Adhere to and every checklist in the existing PR template.
+   4. Not exceed 2000 lines of changes.
```

Comment:
Grammar thing, could you make the sentence work like this:

1. All PRs must
  - be approved ...
  - be approved ...
  - adhere to ...
  - not ...
  
  And make the sublist an unordered list.